bionic/core/flow_execution.py

```diff
@@ -588,5 +587,22 @@ def log_computed_aip(self, task_key):
         self._log("Computed   %s using AIP", task_key)
 
 
+# TODO Consider introducing the term "query" to refer to the scope of one get() call.
```

Comment:
I think `Context` works well for this scope.

Reply:
I think `Context` is a good name for the kind of object it is, but I'm not sure it makes the scope clear. Also, if we have other classes with the same scope, it would be nice if they all had the prefix. (E.g., `TaskCompletionRunner`, `TaskRunnerEntry`, and `ExecutionContext` all have the same scope, so it would be nice if they all started with `Execution` or `Query`. Likewise, `Flow`, `EntityDeriver`, `FlowConfig`, and `TaskState` also share a scope and could also have the same prefix. (`Flow`?))